thoth_user_api/swagger.yaml

```diff
@@ -499,7 +499,7 @@ definitions:
     properties:
       requirements:
         type: string
-        minlength: 2
+        minLength: 2
```

Comment:
This is too much, there are packages that have name that consists of only one letter - see [1]. Anyway, the code you did before can be deleted based on this schema configuration.

[1] https://pypi.org/project/q/